src/v2/Apps/Onboarding/config.ts

```diff
@@ -0,0 +1,70 @@
+import { Engine } from "v2/Utils/Engine"
```

Comment:
We should rename this file / variable to something like `DecisionEngine`

Reply:
@damassi is the current naming too vague?

Reply:
Yep, "engine" could mean a thousand different things

Reply:
Yeah, good point — I vote for `WorkflowEngine` which I think is the more accurate term of art.